src/main/webapp/resources/js/pages/launch/launch-context.js

```diff
@@ -81,20 +108,187 @@ function LaunchProvider({ children }) {
           type: TYPES.LOADED,
           payload: {
             ...details,
+            parameterSet: deepCopy(formattedParameterSets[0]), // This will be the default set of saved parameters
             parameterWithOptions: formattedParameterWithOptions,
+            parameterSets: formattedParameterSets,
           },
         });
       }
     );
   }, [id]);
 
+  function dispatchUseParameterSetById(id) {
+    const set = state.parameterSets.find((p) => p.id === id);
+    dispatch({
+      type: TYPES.PARAMETER_SET,
+      payload: { set: deepCopy(set) },
+    });
+  }
+
   function dispatchLaunch(values) {
     launchPipeline(id, values);
   }
 
+  /**
+   * Dispatch function called when a user modifies the current saved parameter
+   * set parameter values, and wants to use them without saving.
+   *
+   * @param {array} parameters - list of key value pairs for the parameters ({mame: value})
```

Comment:
Typo: change to `name` from `mame`

Reply:
Updated in 7fe76aabd899430c263f776fc61c75da7a697cd7